app/controllers/group_memberships_controller.rb

```diff
@@ -0,0 +1,11 @@
+class GroupMembershipsController < AuthenticationController
+  def create
+    result = CreateGroupMembership.call(user_id:  current_user.id,
+                                        group_id: params[:group_id])
```

Comment:
@Enriikke so here I'm not entirely sure how I'm going to get the `group_id`. Perhaps we could talk a bit about this...